sailor/assetcentral/utils.py

```diff
@@ -429,28 +429,36 @@ def _is_non_string_iterable(obj):
     return isinstance(obj, Iterable)
 
 
+def validate_user_input(input_dict, field_map, forbidden_fields=()):
+    for field_name in forbidden_fields:
+        their_name_put = field_map.get(field_name).their_name_put
```

Comment:
why `.get` and not just `field_map['field_name']`.

Reply:
no reason. I can change it